plugins/lsp/language-servers/default.nix

```diff
@@ -660,10 +663,11 @@ in
   imports = lib.lists.map lspHelpers.mkLsp servers ++ [
     ./ccls.nix
     ./efmls-configs.nix
-    ./nixd.nix
     ./pylsp.nix
     ./rust-analyzer.nix
     ./svelte.nix
     ./vls.nix
   ];
+
+  config = lib.mkMerge [ nixdSettings.config ];
```

Comment:
Why use `mkMerge` here? Couldn't this just be `inherit (nixdSettings) config;`?

Reply:
It could, but this is in case we need the same for other serverq